wicket/src/state/status.rs

```diff
@@ -0,0 +1,141 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This code is all just copied from other files. The only thing that changed is the styling in `LivenessState`.